process/block/sovereignChainBlock.go

```diff
@@ -916,17 +921,31 @@ func (scbp *sovereignChainBlockProcessor) createOutGoingMiniBlockData(outGoingOp
 		OutGoingOperations: outGoingOperationsData,
 	})
 
-	// TODO: We need to have a mocked transaction with this hash to be saved in storage and get rid of following warnings:
-	// 1. basePreProcess.createMarshalledData: tx not found hash = bf7e...
-	// 2. basePreProcess.saveTransactionToStorage  txHash = bf7e... dataUnit = TransactionUnit error = missing transaction
-	// Task for this: MX-14716
 	return &block.MiniBlock{
 		TxHashes:        outGoingOpHashes,
 		ReceiverShardID: core.MainChainShardId,
 		SenderShardID:   scbp.shardCoordinator.SelfId(),
 	}, outGoingOperationsHash
 }
 
+func (scbp *sovereignChainBlockProcessor) addOutGoingTxToPool(outGoingOp *sovCore.OutGoingOperation) {
+	tx := &transaction.Transaction{
+		GasLimit: scbp.economicsData.ComputeGasLimit(
```

Comment:
I think gasLimit should be much higher. not only outGoingOpData - but it needs to be computed from somewhere.

Reply:
As discussed, this is just a dummy stored tx, so actually no gas is used; this value is used just to pass some basic gas checks futher down in code